core/amber/src/main/scala/edu/uci/ics/amber/engine/common/promise/RPCHandlerInitializer.scala

```diff
@@ -0,0 +1,28 @@
+package edu.uci.ics.amber.engine.common.promise
+
+import com.twitter.util.{Future, Promise}
+import edu.uci.ics.amber.engine.common.ambertag.neo.VirtualIdentity.ActorVirtualIdentity
+import edu.uci.ics.amber.engine.common.promise.RPCServer.RPCCommand
+
+import scala.reflect.ClassTag
+
+class RPCHandlerInitializer(rpcClient: RPCClient, rpcServer: RPCServer) {
+
+  def registerHandler[B, C:ClassTag](handler: C => B)(implicit ev: C <:< RPCCommand[B]):Unit = {
```

Comment:
Add a comment here to say that this is a sync handler registration function.  Similarly add a comment below to say that it is an async handler registration function